pkgs/development/tools/misc/blackfire/php-probe.nix

```diff
@@ -129,7 +129,7 @@ stdenv.mkDerivation (finalAttrs: {
         createUpdateable = path: _value:
           lib.nameValuePair
             (createName path)
-            (finalAttrs.self.overrideAttrs (attrs: {
+            (finalAttrs.overrideAttrs (attrs: {
```

Comment:
This will need to be `finalAttrs.finalPackage`.

Reply:
TIL ! Thanks.